crates/miden-objects/src/transaction/executed_tx.rs

```diff
@@ -174,22 +164,14 @@ impl ExecutedTransaction {
     /// Returns individual components of this transaction.
     pub fn into_parts(
         self,
-    ) -> (AccountDelta, TransactionOutputs, TransactionWitness, TransactionMeasurements) {
-        let tx_witness = TransactionWitness {
-            tx_inputs: self.tx_inputs,
-            tx_args: self.tx_args,
-            foreign_account_code: self.foreign_account_code,
-            advice_witness: self.advice_witness,
-        };
-
-        (self.account_delta, self.tx_outputs, tx_witness, self.tx_measurements)
+    ) -> (AccountDelta, TransactionOutputs, TransactionInputs, TransactionMeasurements) {
```

Comment:
nit: I would maybe change the order to:
```
(TransactionInputs, TransactionOutputs, AccountDelta, TransactionMeasurements)
```
This would be more consistent with the order of parameters in the constructor.